phoonnx_train/optispeech/values.py

```
@@ -0,0 +1,169 @@
+        data = self.as_dict()
+        kwargs = {}
+        for name, value in self.as_dict().items():
+            if _TORCH_AVAILABLE and isinstance(value, np.ndarray) or isinstance(value, torch.Tensor):
+                kwargs[name] = torch.as_tensor(value)
+            else:
+                kwargs[name] = value
+        cls = type(self)
+        return cls(**kwargs)
+
+    def as_numpy(self):
+        data = self.as_dict()
```

Ruff (0.15.1)

[error] Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)